src/scripts/components/Input.jsx

```diff
@@ -40,36 +41,36 @@ class Input extends React.Component {
   _handleChange = (event) => {
     this.setState({ value: event.target.value });
 
-    if (this.props.onChange && typeof (this.props.onChange === 'function')) {
-      this.props.onChange(event.target.name, event.target.value.trimLeft());
+    if (this.props.onChange) {
+      this.props.onChange(event.target.name, event.target.value.trimLeft(), event.target.rawValue);
```

Comment:
Do you think it's best to pass in `value` as the second argument instead of `rawValue`? The .value being the Cleave formatted string, whereas rawValue is what is actually in the input.

I'm fine with it either way, but it feels more backwards compatible passing in the rawValue as the second argument here. Just my opinion

Reply:
Ah yep, you're 100% right, good idea I'll update that.